towncrier_template.j2

```diff
@@ -0,0 +1,26 @@
+{% if render_title %}
+## v{{ versiondata.version }} ({{ versiondata.date }})
+
+{% endif %}
+{% for section, _ in sections.items() %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section] %}
+{% if sections[section][category]|length != 0 %}
+### {{ definitions[category]['name'] }}
+
+{% if definitions[category]['showcontent'] %}
+{% for text, values in sections[section][category].items() %}
+- {{ values|join(', ') }} - {{ text }}
+{% endfor %}
+
+{% else %}
+- {{ sections[section][category]['']|join(', ') }}
+
```

Comment:
What is this part doing?

Reply:
This gives you the option to have categories that don't show the summary in the changelog. So if we had a `misc` category instead of looking like this:


### Misc

- [#111](https://github.com/nautobot/nautobot/issues/111) - Change summary 1
- [#222](https://github.com/nautobot/nautobot/issues/222) - Change summary 2


It would look like this

### Misc

- [#111](https://github.com/nautobot/nautobot/issues/111) , [#222](https://github.com/nautobot/nautobot/issues/222)

Changes like "fixed typo" that don't need to be called out in the release notes